ParisBNF/abb/BNFabb195.xml

```diff
@@ -672,7 +674,7 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
             <langUsage><language ident="gez">Gǝʿǝz</language></langUsage>
         </profileDesc>
         <revisionDesc>
-            <change who="DN" when="2019-12-12">Created entity</change>
+            <change who="DN" when="2020-05-12">Created entity</change>
```

Comment:
I think we have August, not that it is so important

I would also add a change element to the edited records

Reply:
ok, i'll do